pkg/domain/infra/abi/images.go

```diff
@@ -336,20 +336,54 @@ func (ir *ImageEngine) Push(ctx context.Context, source string, destination stri
 		pushOptions.Writer = os.Stderr
 	}
 
-	pushedManifestBytes, pushError := ir.Libpod.LibimageRuntime().Push(ctx, source, destination, pushOptions)
-	if pushError == nil {
-		if options.DigestFile != "" {
-			manifestDigest, err := manifest.Digest(pushedManifestBytes)
-			if err != nil {
-				return err
+	pushImage := func(dest string) error {
+		pushedManifestBytes, pushError := ir.Libpod.LibimageRuntime().Push(ctx, source, dest, pushOptions)
+		if pushError == nil {
+			if options.DigestFile != "" {
```

Comment:
What does `DigestFile` + `all-tags` do? The implemented behavior is not useful AFAICT.